packages/react/src/components/CTA/__stories__/CTA.stories.js

```diff
@@ -53,8 +53,23 @@ storiesOf('Components|CTA', module)
               href: urlBy[type],
             },
             image: {
-              defaultImage: 'https://picsum.photos/id/672/672',
-              alt: 'featured card image',
+              images: [
+                {
+                  src: 'https://dummyimage.com/320x160/ee5396/fff&text=2x1',
+                  minWidth: 'sm',
+                },
+                {
+                  src: 'https://dummyimage.com/400x400/ee5396/fff&text=1x1',
```

Comment:
nitpicky thing, should we do `1:1` or `16:9` instead of `1x1` and `16x9`?

Reply:
I agree, aspect ratio is usually shown as `1:1`. @wonilsuhibm are you ok with this? i used `16x9` based on your earlier example.

Reply:
@kennylam I'd say let's just go ahead and change it